spec.md

```diff
@@ -3503,6 +3558,18 @@ The lowering algorithm for the scalarized convention operates as follows:
    suffix, `_<name>`, to the field called `name`. Fields are scalarized
    recursively, depth-first, and left-to-right.
 
+5. Enumeration-typed ports are scalarized to a tag port and a value port. The
+   tag port is appended with the suffix `_tag` and is a `UInt` type with the
+   minimum number of bits needed to represent the tags of all members of the
+   enumeration type.  The data port is appended with the suffix `_data` and is
+   a `UInt` with the minimum number of bits needed to represent the largest
+   data of all members of the enumeration type. Each member of the enumeration
+   has its data mapped to the lowered data port starting at the most
+   significant byte and taking up as many bits as they are wide.  Aggregate
+   data is mapped recursively, depth-first, and left-to-right to contiguous
+   bits of the data port. Enumeration data maps the tag followed by the value
+   port.
```

Comment:
Using `type` instead of `member` would be clearer. (Is that right?)

The PR mentions "byte" when I think it means "bit".

Could the PR show an example of the lowering to make it obvious?